wandb/fastai/__init__.py

```diff
@@ -43,23 +45,27 @@ class WandbCallback(TrackerCallback):
 
     def __init__(self,
                  learn,
-                 log=None,
-                 show_results=False,
-                 save_model=False,
+                 log="all",
```

Comment:
I think we should set this to gradients by default like `wandb.watch`

Reply:
Is there a specific reason behind it? I would assume the more data we log, the better… unless it affects performance

Reply:
We chose gradients because there’s added bandwidth and compute on the client and server for both.  Felt like the default should add the most value for least cost.